canonical-web-and-design/jaas-dashboard
# Remove multiple scroll bars from slide panels

## Done

Remove multiple scroll bars from slide panels

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Go to model details page
- Click on a table row to open slide panels
- Verify that multiple scrollbars no longer appear

## Details

Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/826
